src/VisualStudio/Core/Def/Implementation/TableDataSource/VisualStudioBaseDiagnosticListTable.LiveTableDataSource.cs

```diff
@@ -35,7 +35,7 @@ internal abstract partial class VisualStudioBaseDiagnosticListTable
         /// See <see cref="VisualStudioDiagnosticListTableWorkspaceEventListener.VisualStudioDiagnosticListTable.BuildTableDataSource"/>
         /// for error list diagnostic source for "Build only" setting.
         /// </summary>
-        protected class LiveTableDataSource : AbstractRoslynTableDataSource<DiagnosticTableItem>
+        protected class LiveTableDataSource : AbstractRoslynTableDataSource<DiagnosticTableItem, DiagnosticsUpdatedArgs>
```

Comment:
`DiagnosticsUpdatedArgs` is still a wonky name.  can maybe remove that in the future as well.